src/components/MenuSectionPage.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect} from 'react';
+import Merchant from './Merchant';
+
+
+
+function MerchantList() {
+  const [items, setItems] = useState([]);
+  const initialParagraph=["套餐","主食","甜點","飲料"];
+  useEffect(() => {
+    setItems(Array.from({ length: 4 }).map((_, index) => ({
+      type: 'paragraph',
+      text: `${initialParagraph[index]}`,
+      merchants: Array.from({ length: 5 }).map((_, i) => ({
+            id: i+1,
+            name: `新商家 ${i+1}`,
+            distance: (Math.random() * 10).toFixed(1),
+            costDownLimit: Math.floor(Math.random() * 100),
+            costUpLimit: Math.floor(Math.random() * 200),
+            starRate: (Math.random() * 5).toFixed(1),
+            starNumber: Math.floor(Math.random() * 100),
+      })),
+    }))
+  );
+  }, []);
+  return (
+    <div className="min-h-screen flex flex-col items-center justify-center container mx-auto p-4">
+      {items.map((item, index) => (
+        <div key={index}>
+          <p className="text-lg font-medium my-4">{item.text}</p>
+          {item.merchants.map((merchant, i) => (
```

Comment:
You missing the `key` attribute when use `map` to render list data.
this is very import to add `key`!!